### PR TITLE
Layout tweaks

### PR DESCRIPTION
**Pull request by NickM-27**
- Fix review items and no review items background showing
- Fix -1 text on count
- Clean up spacing for mobile recordings view
- no preview text when preview is not found

### CODE REVIEW

File: web/src/views/events/EventView.tsx [hunk 1]
```diff
@@ -441,10 +452,16 @@ function DetectionReview({
           />
         )}
 
-        {itemsToReview == 0 && (
-          <div className="size-full flex flex-col justify-center items-center">
+        {currentItems == null && (
```
Comment (hawkeye217) [inline web/src/views/events/EventView.tsx:455]: could this just be `{!currentItems && ( ...`?

File: web/src/views/events/EventView.tsx [hunk 2]
```diff
@@ -441,10 +452,16 @@ function DetectionReview({
...
+          <div className="absolute left-1/2 -translate-x-1/2 top-1/2 -translate-y-1/2">
+            <ActivityIndicator />
+          </div>
+        )}
+
+        {currentItems != null && currentItems.length == 0 && (
```
Comment (hawkeye217) [inline web/src/views/events/EventView.tsx:461]: could this just be `{currentItems?.length === 0 && (`

`===` is for a strict equality comparison making sure the length is exactly zero.

File: web/src/views/events/EventView.tsx [hunk 3]
```diff
@@ -479,17 +496,19 @@ function DetectionReview({
...
-              <Button
-                className="text-white"
-                variant="select"
-                onClick={markAllReviewed}
-              >
-                Mark these items as reviewed
-              </Button>
-            </div>
-          )}
+          {currentItems != null &&
```
Comment (hawkeye217) [inline web/src/views/events/EventView.tsx:499]: could this just be `{currentItems?.length > 0 && itemsToReview > 0 && (`? Maybe I'm missing something that you're checking explicitly for?

File: web/src/views/events/EventView.tsx (hunk 1 above)
Comment (NickM-27) [reply web/src/views/events/EventView.tsx:455]: !current items covers empty list as well I believe

Comment (NickM-27) [reply web/src/views/events/EventView.tsx:455]: never mind looks like it doesn't, so that works out

**Review by hawkeye217 — APPROVED** [review]

**Pull request merged**